README.md

```diff
@@ -422,6 +422,10 @@ To learn how to get notifications when resource 1 changes, or how to use resourc
 ## Known issues
 
 1. Mutex issue using debug profile, issue #[303](https://github.com/ARMmbed/mbed-os-example-client/issues/303).
+1. UDP connection fails with on-line compiler (ARM CC 5.06 update 3)#[357](https://github.com/ARMmbed/mbed-os-example-client/issues/357)
+1. K64F LWIP IPv6 not working with ARM CC 5.06 update 3 (4 or 6)#[359](https://github.com/ARMmbed/mbed-os-example-client/issues/359).
```

Comment:
What does that (4 or 6) mean?

Reply:
If its IPv4 or IPv6 then please be verbal about it.
Now it looks like ARM CC 5.06 update 3, update 4 or update 6. And that sentence is untrue because update 6 works.

Reply:
You're right, should 4,5 but I'll reword it.